config/locales/en-GB.yml

```diff
@@ -1311,8 +1311,8 @@ en-GB:
         is accurate and up to date.
       community_driven_title: Community Driven
       community_driven_html: |-
-        OpenStreetMap's community is diverse, passionate, and growing every day. Our contributors include enthusiast mappers, GIS professionals, engineers running the OSM servers, humanitarians mapping disaster-affected areas, and many more.
```

Comment:
From CONTRIBUTING.md

> If you make a change that involve the locale files (in config/locales) then please only submit changes to the en.yml file. The other files are updated via Translatewiki and should not be included in your pull request.

Reply:
Thanks. Maybe this could be an automated check? https://github.com/openstreetmap/openstreetmap-website/pull/2740 is a nudge in this direction that might be interesting.